pkg/rancher-desktop/pages/marketplace/details.vue

```diff
@@ -0,0 +1,352 @@
+        this.metadata?.LatestVersion.Labels[
+          'com.docker.extension.detailed-description'
+        ].replace('h1', 'h2') || ''
+      );
+    },
+
+    getPreviewImage() {
+      if (!this.metadata) {
+        return;
+      }
+
+      // TODO: Maybe we want to use a gallery here ?
+      const screenshots =
+        this.metadata?.LatestVersion.Labels['com.docker.extension.screenshots'];
+
+      return JSON.parse(screenshots)[0].url || '';
+    },
+  },
+
+  watch: {},
+
+  mounted() {
+    if (!this.extension) {
+      this.$router.push('/marketplace');
+    }
+
+    // TODO: get metadata from the backend
+    this.metadata = demoMetadata[this.extension];
+
+    if (!this.metadata) {
+      return;
+    }
+
+    this.$store.dispatch('page/setHeader', {
+      title:
+        this.metadata?.LatestVersion.Labels['org.opencontainers.image.title'] ||
+        this.extension,
+    });
+
+    this.extensionDetails = {
+      name:
+        this.metadata?.LatestVersion.Labels['org.opencontainers.image.title'] ||
+        this.extension,
+      icon:
+        this.metadata?.LatestVersion.Labels[
+          'com.docker.desktop.extension.icon'
+        ] || '',
+      links:
+        JSON.parse(
+          this.metadata?.LatestVersion.Labels[
+            'com.docker.extension.additional-urls'
+          ],
+        ) || [],
+      version:    this.metadata?.LatestVersion.Tag || [],
+      platforms:  this.metadata?.LatestVersion.Platforms || [],
+      categories: this.metadata?.Categories || [],
+      lastUpdate: this.metadata?.UpdatedAt || [],
+    };
+  },
+
+  methods: {
+    resetBanners() {
+      this.loading = true;
+      this.error = null;
+      this.response = null;
+    },
+
+    appInstallation(action = 'install') {
+      this.resetBanners();
+
+      fetch(
+        `http://localhost:${ this.credentials?.port }/v1/extensions/${ action }?id=${ this.extension }`,
+        {
+          method:  'POST',
+          headers: new Headers({
+            Authorization: `Basic ${ window.btoa(
+              `${ this.credentials?.user }:${ this.credentials?.password }`,
+            ) }`,
+            'Content-Type': 'application/x-www-form-urlencoded',
+          }),
+        },
+      ).then((r) => {
+        if (!r.ok) {
+          this.error = r.statusText;
+          this.loading = false;
+        }
+
+        if (r.status === 201) {
+          this.response = this.t(`marketplace.banners.${ action }`);
+
+          this.loading = false;
+
+          if (action === 'uninstall') {
+            this.isInstalled = false;
+          } else {
+            this.isInstalled = true;
+          }
+        }
+      });
+    },
+
+    formatDate(dateString) {
+      const date = new Date(dateString);
+      const day = date.getDate().toString().padStart(2, '0');
+      const month = (date.getMonth() + 1).toString().padStart(2, '0');
+      const year = date.getFullYear();
+
+      return `${ day }/${ month }/${ year }`;
+    },
```

Comment:
We could probably make use of `dayjs` to distill this down to something along the lines of

```suggestion
function formatDate(dateString) {
  const date = dayjs(dateString);
  return date.format('DD/MM/YYYY');
}```